x/bank/types/query.pb.go

```diff
@@ -461,10 +460,10 @@ type QueryClient interface {
 }
 
 type queryClient struct {
-	cc grpc1.ClientConn
```

Comment:
i ran `make proto-gen` as instructed by the contributing. It seems there is a gap in the documentation for this